lib_xcore_math/CMakeLists.txt

```diff
@@ -36,14 +37,24 @@ target_include_directories( ${LIB_NAME}
 
 target_compile_options( ${LIB_NAME}
   PRIVATE
-    -Os -g
-    -Wall
-    -Wextra
-    -Werror
+    -Os
     $<$<PLATFORM_ID:XCORE_XS3A>:-Wno-xcore-fptrgroup>
 )
 
-
+# Add options for different compilers
+if ( NOT CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  target_compile_options(${LIB_NAME}
+    PRIVATE
+      -g
+      -Wno-unused-variable
+      -Wno-format
```

Comment:
we don't need these

Reply:
I removed them. Let's see what Jenkins think about it.